addons/mail/static/src/components/notification_list/notification_list.xml

```diff
@@ -39,7 +39,7 @@
                             />
                         </t>
                         <t t-if="!notificationView_last">
-                            <div class="o_NotificationList_separator border-bottom"/>
+                            <div class="o_NotificationList_separator flex-grow-0 flex-shrink-0 border-bottom"/>
```

Comment:
The default value of `flex-grow` is `0`, so it's not necessary to add `flex-grow-0`.